src/python/xrt_binding.py

```diff
@@ -989,9 +1079,9 @@ def xclRgWrite(handle, cu_index, offset, data):
     :param data: Pointer to where result will be saved
     :return: 0 or appropriate error number
     """
-    libc.xclRegRead.restype = ctypes.c_int
-    libc.xclRegRead.argtypes = [xclDeviceHandle, ctypes.uint32_t, ctypes.uint32_t, ctypes.c_uint32_t]
-    return libc.xclRegRead(handle, cu_index, offset, data)
+    libcore.xclRegRead.restype = ctypes.c_int
+    libcore.xclRegRead.argtypes = [xclDeviceHandle, ctypes.uint32_t, ctypes.uint32_t, ctypes.c_uint32_t]
+    return libcore.xclRegRead(handle, cu_index, offset, data)
```

Comment:
Oops, this can never have been tested.  Should be xclRegWrite, and why is the function called "xclRgWrite" with missing 'e'?

Reply:
Fixed